src/js/views/patients/sidebar/patient/patient-sidebar_views.js

```diff
@@ -0,0 +1,50 @@
+import Radio from 'backbone.radio';
+import { View } from 'marionette';
+
+import 'sass/modules/buttons.scss';
+import 'sass/modules/forms.scss';
+import 'sass/modules/textarea-flex.scss';
+import 'sass/modules/sidebar.scss';
+
+import PatientSidebarTemplate from './patient-sidebar.hbs';
+
+import { WidgetCollectionView } from 'js/views/patients/widgets/widgets_views';
+
+import './patient-sidebar.scss';
+
+const LayoutView = View.extend({
+  childViewTriggers: {},
+  className: 'sidebar flex-region',
+  template: PatientSidebarTemplate,
+  regions: {
+    widgets: '[data-widgets-region]',
+  },
+  triggers: {
+    'click .js-close': 'close',
+    'click .js-patient': 'click:patient',
+  },
+  templateContext() {
+    return {
+      firstName: this.patient.get('first_name'),
+      lastName: this.patient.get('last_name'),
+    };
+  },
+  initialize({ patient, widgets }) {
+    this.patient = patient;
```

Comment:
This'd be a little simple if the patient was passed as the model.  Then you also wouldn't need `templateContext`.  The `onRender` would just pass the model down and although attaching `widgets` here is perfectly fine, since it's single use, you wouldn't need the `initialize` and could just do `collection: this.getOption('widgets'),` in the `onRender`